src/main/java/seedu/address/logic/commands/MoveCommand.java

```diff
@@ -0,0 +1,106 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.pdf.Directory;
+import seedu.address.model.pdf.Pdf;
+import seedu.address.model.pdf.exceptions.DuplicatePdfException;
+
+import java.io.File;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PDFS;
+
+/**
+ * Moves a PDF listed in PDF++ to another specified location.
+ */
+public class MoveCommand extends Command {
+
+    public static final String COMMAND_WORD = "move";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Moves one specified pdf identified "
+            + "by the index number used in the displayed pdf list.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[NEW_DIRECTORY]\n"
+            + "Example: " + COMMAND_WORD + " 1 Desktop\\main\\target";
```

Comment:
Should we put an actual location like "C:\Users\....\main..\target" instead of  "Desktop\\main\\target"